.github/workflows/build.yml

```diff
@@ -92,16 +92,19 @@ jobs:
             LIBOMP_NUM_HIDDEN_HELPER_THREADS=0
             KMP_DUPLICATE_LIB_OK=TRUE
             OMP_NUM_THREADS=1
-            MACOSX_DEPLOYMENT_TARGET=14.0
+            MACOSX_DEPLOYMENT_TARGET=${{ matrix.os == 'macos-13' && '13.0' || '14.0' }}
           CIBW_ENVIRONMENT_WINDOWS: >
             FAISS_OPT_LEVEL=${{ matrix.opt_level }}
-            CMAKE_PREFIX_PATH="c:\\opt"
-            LIB="${LIB};${CMAKE_PREFIX_PATH}\\lib;${CONDA}\\Library\\lib"
-            CPATH="${CPATH};${CMAKE_PREFIX_PATH}\\include;${CONDA}\\Library\\include"
-          CIBW_BEFORE_ALL: bash scripts/build_${{ runner.os }}.sh
+            CMAKE_PREFIX_PATH=${{ matrix.os == 'windows-11-arm' && '"c:\\opt\\OpenBLAS"' || '"c:\\opt"' }}
```

Comment:
This part seems to append `OpenBLAS` directory because the zip contents are expanded with this hierarchy. Would it be possible to move the contents from the zip destination one level above in the bash script to avoid all these conditions?